common/lib/capa/capa/inputtypes.py

```diff
@@ -85,6 +85,7 @@ def __init__(self, status, gettext_func=unicode):
         names = {
             'correct': _('correct'),
             'incorrect': _('incorrect'),
+            'partially-correct': _('partially-correct'),
```

Comment:
Ok. Who programmed this part originally, so I can ask them what's going on here and whether it's user-facing?